menpo/model/pca.py

```diff
@@ -134,12 +134,9 @@ def _constructor_helper(self, eigenvalues, eigenvectors, mean, centred,
             MeanLinearVectorModel.__init__(self, eigenvectors,
                                            np.zeros(mean.shape, dtype=mean.dtype))
         self.centred = centred
-        self._eigenvalues = eigenvalues
-        # start the active components as all the components
-        self._n_active_components = int(self.n_components)
-        self._trimmed_eigenvalues = np.array([])
+        self._original_eigenvalues = eigenvalues
         if max_n_components is not None:
```

Comment:
Also, max_n_components can be a float because of the n_active_components setter. However, in the initial documentation it just mentions the 'int', so you might want to either catch the float (allow only int) or extend the documentation to 'float' case.